pkg/controller/deployment/util/deployment_util_test.go

```diff
@@ -960,19 +960,19 @@ func TestDeploymentComplete(t *testing.T) {
 		expected bool
 	}{
 		{
-			name: "complete",
+			name: "not complete #0",
```

Comment:
Suggest making it more descriptive, something like "not complete: min but not all pods become available"